Dockerfile.load

```diff
@@ -3,9 +3,9 @@ LABEL maintainer="Caleb Dunn"
 WORKDIR /app
 COPY requirements.txt .
 RUN pip install -r requirements.txt
-COPY . /app
+COPY ./app /app
```

Comment:
kinda nitpicky but this can be rewritten as `COPY ./app .` The `WORKDIR /app` sets the containers working directory to `/app` so `.` will copy to the working directory.